kpatch/kpatch

```diff
@@ -158,6 +165,8 @@ load_module () {
 			echo "loading core module: $COREMOD"
 			insmod "$COREMOD" || die "failed to load core module"
 		fi
+
+		init_sysfs_var
```

Comment:
I think this call to `init_sysfs_var` is redundant as the one below should have already been called via `load_module` -> `verify_module_checksum`, right?

Reply:
The `init_sysfs_var()` call on line 234 is called first, no matter what the subcommand is.  However, at that point the core module (kpatch.ko) might not have been loaded yet and SYSFS might have the wrong value.  That's why `init_sysfs_var()` is called again after loading the core module.

Does that answer your question?  Would a comment help?

Reply:
@joe-lawrence ping?